### PR TITLE
Events

### PR DESCRIPTION
**Pull request by ahzia**
In this milestone, I have worked on adding functionalities to calculator buttons.
Kindly review it and share your feedback with me.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,40 +1,74 @@
...
           <div className="d-flex">
-            <div className="btn-rectingle btn-zero d-flex justify-content-center align-items-center border">0</div>
-            <div className="btn-rectingle d-flex justify-content-center align-items-center border">.</div>
-            <div className="btn-rectingle d-flex justify-content-center align-items-center border bg-temp">=</div>
+            <div id="0" role="button" className="btn-rectingle btn-zero d-flex justify-content-center align-items-center border">0</div>
+            <div id="." role="button" className="btn-rectingle d-flex justify-content-center align-items-center border">.</div>
+            <div id="=" role="button" className="btn-rectingle d-flex justify-content-center align-items-center border bg-temp">=</div>
           </div>
         </div>
       </div>
```
Comment (wandji20) [inline src/components/Calculator.js]: - [x] It will be nice to create children components and implement state lifting techniques to pass props as desired.

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @ahzia 
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for this project.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark: Descriptive commit messages

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Optional comments are not required to mark this project approved
- [x] Some calculations are buggy (eg. division, modulus). Please verify your calculator is fully functional as required.
error


**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahzia** [comment]
@wandji20  Thank you, for the review, I implemented the request changes.

**Review by DeliceLydia — APPROVED** [review]
# Approved🎉 🎉 

## Hi @ahzia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional
- [x] Consider adding a default zero in the screen of your calculator


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahzia** [comment]
@DeliceLydia Thank you for the Review

**Pull request merged**